src/components/Pages/Guide/_Guide.scss

```diff
@@ -79,6 +79,18 @@ $max-guide-tablet-width: $coa-medium-screen;
   overflow-y: scroll;
   margin-top: $sticky-guide-top;
   top: 0px;
+
+  // Hide TOC scrollbar in:
+  // Webkit eg. Chrome/Safari
+  &::-webkit-scrollbar {
+    width: 0 !important;
+  }
+
+  // Edge/Internet Explorer
+  -ms-overflow-style: none;
+
+  // Firefox
+  scrollbar-width: none;
```

Comment:
tested in FF and found this was definitely the thing making it work. There was probably some scss magic happening before to make it work but now it's explicit.